encoding/fuzz_test.go

```diff
@@ -410,3 +411,4 @@ func (s *byteSource) Int63() int64 {
 }
 
 func (s *byteSource) Seed(seed int64) {}
+*/
```

Comment:
Deliberate?

Reply:
The tests need to be adapted to the new interfaces, they aren't valid anymore but I think they are useful to keep around as baseline to be reworked in a follow up PR.

Reply:
https://github.com/segmentio/parquet-go/pull/174/files